packages/kbn-ui-framework/src/components/form/check_box/_check_box.scss

```diff
@@ -1,6 +1,7 @@
 $checkboxBackgroundColor: #FFF;
 $checkboxBorderColor: #BEBEBE;
 $checkboxIconColor: #FFF;
+$checkboxDisaledColor: #D9D9D9;
```

Comment:
Spelling error here: missing the 'b' in 'Disabled'

Reply:
Good catch, thanks :)